esp/src/src-react/layouts/MessageBox.tsx

```diff
@@ -56,7 +56,7 @@ export const MessageBox: React.FunctionComponent<MessageBoxProps> = ({
 
     const close = React.useCallback(() => setShow(false), [setShow]);
 
-    return <Modal isOpen={show} onDismiss={close} isModeless={true} dragOptions={dragOptions}
+    return <Modal isOpen={show} onDismiss={close} isModeless={false} dragOptions={dragOptions}
```

Comment:
Changing this modeless prop to true might have been for a specific request that allowed selecting text behind the Modal? Eg, copying a WU# to paste in a filter. (https://github.com/hpcc-systems/HPCC-Platform/pull/15487)

Reply:
I will have to take another look as bumping the @fluentui/react version with this set to true is breaking all the MessageBox instances...

Reply:
Ok - found reference(s) to the issue and a workaround

Reply:
The workaround was to allow for isModeless=true?

Reply:
I have renabled it now.